Src/VimCore/VimTextBuffer.fs

```diff
@@ -23,6 +23,8 @@ type internal VimTextBuffer
     let _switchedModeEvent = StandardEvent<SwitchModeKindEventArgs>()
     let mutable _modeKind = ModeKind.Normal
     let mutable _lastVisualSelection : ITrackingVisualSelection option = None
+    let mutable _lastModifiedSpan : ITrackingVisualSelection option = None

```

Comment:
This doesn't appear to be used (may just be missing it in the change).  If not can we delete?